### PR TITLE
feat(wigtobigwig): Added docker. For Fix #1327

### LINKED ISSUE #1327: wigToBigWig
**Is your feature request related to a problem? Please describe.**
Large wig's load slowly in igv. Convert wigs for display to bigwig (rhocall, TIDDIT).
Similarly with the largest bed files. UPD-sites-bed would thus need conversion to bigBED.

**Describe the solution you'd like**
Use wigToBigWig or similar to convert wig files into bigwig, and bed to bigbed with bedToBigBed.

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- Adds a docker for wigtobigwig

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: templates/mip_install_rd_dna_config_-1.0-.yaml [hunk 1]
```diff
@@ -155,6 +155,10 @@ singularity:
     executable:
       vt:
     uri: docker://clinicalgenomics/vt:2015.11.10
+  wigtobigwig:
+    executable:
+      wigtobigwig:
```
Comment (jemten) [inline templates/mip_install_rd_dna_config_-1.0-.yaml]:
```suggestion
      wigToBigWig:
```

Comment (jemten) [reply templates/mip_install_rd_dna_config_-1.0-.yaml]: I was to late 😄

Comment (henrikstranneheim) [reply templates/mip_install_rd_dna_config_-1.0-.yaml]: 😄

**Comment by henrikstranneheim** [comment]
Yeah, sorry I changed my mind and decided to install all ucsc tools in the same container.

**Comment by henrikstranneheim** [comment]
I will remove wigToBigWig from the executables in the rseqc image. rseqc will still install it in the image, but there will only be acces through the proxy bin to ucsc container wigToBigWig. 

This was what you meant right?

**Comment by jemten** [comment]
Exactly

**Pull request merged**